### PR TITLE
Add crashlytics

### PR DESCRIPTION
**Pull request by dorshay6**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/ios/Locomotion.xcodeproj/xcshareddata/xcschemes/Locomotion.xcscheme [hunk 1]
```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (avivbarniv) [inline examples/client/Locomotion/ios/Locomotion.xcodeproj/xcshareddata/xcschemes/Locomotion.xcscheme:44]: change to Debug

File: examples/client/Locomotion/src/services/network.js [hunk 2]
```diff
@@ -72,6 +73,7 @@ class Network {
         this.axios.defaults.headers.common['x-loco-ds-id'] = Config.OPERATION_ID;
         this.axios.defaults.headers.common['x-loco-op-id'] = Config.OPERATION_ID;
         return this.axios[method](...args).catch((e) => {
+          crashlytics().log(`HTTP Request Error ${e.message}`);
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/services/network.js:76]: add status code

**Review by avivbarniv — CHANGES_REQUESTED** [review]

File: examples/client/Locomotion/src/services/network.js (hunk 2 above)
Comment (dorshay6) [reply examples/client/Locomotion/src/services/network.js:76]: its in the message

File: examples/client/Locomotion/ios/Locomotion.xcodeproj/xcshareddata/xcschemes/Locomotion.xcscheme (hunk 1 above)
Comment (dorshay6) [reply examples/client/Locomotion/ios/Locomotion.xcodeproj/xcshareddata/xcschemes/Locomotion.xcscheme:44]: will fix in my next pr

**Pull request merged**